editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -66,12 +176,164 @@ export const ClassNameSelect: React.FunctionComponent = betterReactMemo('ClassNa
           value: name,
         }))
 
+  const ChromaThemePrimary = React.useMemo(() => chroma(theme.primary.value), [theme])
```

Comment:
a couple of months ago we bumped eslint, and now it understands property access like this:

 
```suggestion
  const ChromaThemePrimary = React.useMemo(() => chroma(theme.primary.value), [theme.primary.value])
```

since we only use that one value, why rely on the referential stability of `theme` itself

Reply:
I can use this for the data too

Reply:
right!